tests/framework/installer/ceph_installer.go

```diff
@@ -45,7 +45,9 @@ const (
 	// test with the latest nautilus build
 	nautilusTestImage = "ceph/ceph:v14"
 	// test with the latest octopus build
-	octopusTestImage  = "ceph/ceph:v15"
+	octopusTestImage = "ceph/ceph:v15"
+	// test with the latest master image
+	masterTestImage   = "ceph/daemon-base:latest"
```

Comment:
there is also a `ceph/daemon-base:latest-master` tag

Reply:
Thanks @sebastian-philipp, it seems that the daily buid image from master is with the tag "ceph/daemon-base:latest-master-devel"